Source/WebKit/UIProcess/BackingStore.h

```diff
@@ -25,21 +25,26 @@
 
 #pragma once
 
-#if USE(GRAPHICS_LAYER_WC) || (USE(CAIRO) && !PLATFORM(WPE))
+#if USE(GRAPHICS_LAYER_WC) || PLATFORM(GTK) || (USE(CAIRO) && !PLATFORM(WPE))
```

Comment:
After thinking about this a bit more this could be simplified to just
```suggestion
#if !PLATFORM(WPE))
```